x/qgb/client/verify.go

```diff
@@ -157,7 +157,7 @@ func sharesCmd() *cobra.Command {
 	command := &cobra.Command{
 		Use:   "shares <height> <start_share> <end_share>",
```

Comment:
[question] are <start_share> and <end_share> share indexes?

Reply:
yes